### PR TITLE
Add core java dependency + maven deploy

### PR DESCRIPTION
**Pull request by rcbgr**
Add core java dependency + maven deploy

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,48 +1,80 @@
...
+		<name>Coinbase International Exchange Java SDK</name>
+		<description>Sample Java SDK for the Coinbase International Exchange REST APIs</description>
+		<version>0.1.1</version>
+		<url>https://github.com/coinbase-samples/intx-sdk-java</url>
 
     <licenses>
         <license>
             <name>Apache License, Version 2.0</name>
-            <url>https://www.apache.org/licenses/LICENSE-2.0</url>
+						<url>https://github.com/coinbase-samples/intx-sdk-java/blob/main/LICENSE</url>
```
Comment (cfluke-cb) [inline pom.xml:33]: looks like some tab vs spaces issue here

File: pom.xml [hunk 2]
```diff
@@ -1,48 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cfluke-cb) [inline pom.xml:1]: this file needs a lint or formatter

**Review by cfluke-cb — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (rcbgr) [reply pom.xml:33]: Agreed. It renders ok in VIM, but I will address in a subsequent PR. We need these approved before the STP meeting tomorrow.

File: pom.xml (hunk 2 above)
Comment (rcbgr) [reply pom.xml:1]: Updated with
```
xmllint --format -
```

File: pom.xml (hunk 1 above)
Comment (rcbgr) [reply pom.xml]: Actually, I fixed

**Review by nmorgan-cb — APPROVED** [review]

**Review by jc-cb — APPROVED** [review]

**Review by cfluke-cb — APPROVED** [review]

**Review by KMORGAN008 — APPROVED** [review]

**Comment by KMORGAN008** [comment]
53ca4e2a83d5ef988c6c5b68c40deabd92fc0bc6

**Comment by KMORGAN008** [comment]
53ca4e2a83d5ef988c6c5b68c40deabd92fc0bc6

**Pull request merged**